pkg/controller/pkg/revision/hook.go

```diff
@@ -80,7 +82,14 @@ func (h *ProviderHooks) Pre(ctx context.Context, pkg runtime.Object, pr v1alpha1
 	if pr.GetDesiredState() != v1alpha1.PackageRevisionInactive {
 		return nil
 	}
-	s, d := buildProviderDeployment(pkgProvider, pr, h.namespace)
+	var controllerConfig *v1alpha1.ControllerConfig
```

Comment:
Ditto my long-var-names-make-me-sad nit from above, especially here where these variable only live for ~5 lines.